test/mosaic-integration-tests/src/test/java/org/eclipse/mosaic/test/SendAndReceiveIT.java

```diff
@@ -208,8 +207,8 @@ public void roundTripMessageTakesRightAmountOfTime() throws Exception {
         long timeOfSending = 310 * TIME.SECOND;
         long delayTmcUpload = 50 * TIME.MILLI_SECOND;
         long delayTmcDownload = 50 * TIME.MILLI_SECOND;
-        long delayVehUpload = 106 * TIME.MILLI_SECOND;
-        long delayVehDownload = 57 * TIME.MILLI_SECOND;
+        long delayVehUpload = 86 * TIME.MILLI_SECOND; //TODO check those values, are those random?
```

Comment:
I suppose they are. I guess this is best fixed by changing the delays of the region the vehicles are located in.
Though this could break other tests 😓